xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties":{
+    "facility": {
+      "type": "string",
+      "description": "Facility Name (NSLS-II, APS, etc)"
+    },
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength in Angstrom, Depreciation Warning: this will depreciated in upcoming releases, please do not use this going forward"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
```

Comment:
is this how we do it?  It is a bit ugly.  I have deja vu though....I think this flat (i.e., not [[first1,last1],[first2,last2]]) is better for searching?